CHANGELOG/CHANGELOG-v0.4.md

```diff
@@ -0,0 +1,23 @@
+# Changelog since v0.3.0
+All notable changes to this project will be documented in this file.
+
+## v0.4.0
+
+### New Features 
+* Support managedClusterAddon registration and maintain the lease of managedClusterAddon.
+
+### Added
+* Upgrade CRD to v1 and k8s lib to v0.21.0-rc.0.
+* Upgrade Go to 1.16.
+* Support DNS names for client cert.
+
+### Changes
+* Merge managed cluster capacity status.
+* Reset csr name only if csr sync failed or secret saved.
```

Comment:
- Use `livez` API instead of `readyz` API to check the status of managed cluster kube-apiserver
- Use `clusterrole` instead of `role` for managed cluster add-ons to reduce number of hub resources

Reply:
done